codegen/protocol-test-codegen-local/build.gradle.kts

```diff
@@ -47,6 +47,10 @@ val codegenTests = listOf(
     CodegenTest(
         "aws.protocoltests.eventstream#RPCTestService",
         "RPCEventStream"
+    ),
+    CodegenTest(
+        "aws.endpointtests.stringarray#EndpointStringArray",
+        "StringArrayEndpointParam"
```

Comment:
Add the endpoint string array Smithy model to list of models used by this build script